### ISSUE TITLE
Secure storing of credentials

### ISSUE THREAD

**Issue by nvuillam** [test-zeus-ai/testzeus-hercules#12]
Credentials should not be stored in a repository

Would it be possible to provide them:
- using ENV variables (they could be defined on CI/CD config)
- from a .env file that would be locally defined but not committed thanks to .gitignore

Thanks :)

**Comment by generalrobin** [comment]
Yes @nvuillam . This can be easily added with a small precommit hook and we will look into the same.

**Comment by nvuillam** [comment]
What do you want to add in a pre-commit hook ? :)
We need to not commit such values 😋

**Comment by shriyanshagnihotri** [comment]
@nvuillam so we have a custom script that can pull the data from AWS KMS, Google secrets and create a test_data secret file on the fly during the execution. @generalrobin is trying to mention the same that this script can be called by the hook in the CI/CD before actually doing the test case execution, in such cases we don't have to put secrets in the flat file.

The script and integration is still under development.

**Comment by nvuillam** [comment]
@shriyanshagnihotri that's great :)
But we also need more simple ways to handle that locally ith not commited .env file, and with ENV variables for CI (remember: Salesforce people are less tech than others 😋 )
